cmake/GenerateVersion.cmake

```diff
@@ -19,30 +13,29 @@ execute_process(
   RESULT_VARIABLE RES
 )
 
-if("${RES}" EQUAL 1)
+if(NOT GIT_TAG)
+  message(DEBUG "Git tag extraction failed with \n"  "   ${ERR}" )
   if(EXISTS ${OUTPUT})
-    message(STATUS "Unable to extract version-string from git: keeping the last known version")
+    message(VERBOSE "Unable to extract version-string from git: keeping the last known version")
```

Comment:
Why set this at VERBOSE? Missing git is the _uncommon_ case. Please, always think about the common case. Users rarely run cmake at "verbose" level, and the common case is that users have git. If we reach this branch there should be a message.